### PR TITLE
docs: fix links url

### PR DESCRIPTION
**Pull request by devagja**
https://github.com/pmndrs/valtio/issues/689

### CODE REVIEW

File: docs/api/basic/proxy.mdx [hunk 1]
```diff
@@ -59,7 +59,7 @@ authState.user.name = 'Nina'
 
 #### Promises in proxies
 
-See [`async`](/docs/advanced/async) for more details.
+See [`async`](/docs/guides/async) for more details.
```
Comment (dai-shi) [inline docs/api/basic/proxy.mdx:62]: Can this be
```suggestion
See [`async`](../../guides/async.mdx) for more details.
```

**Review by dai-shi — COMMENTED** [review]
Thanks for the catch.
We want to be able to browse docs in github.
Can you make all links relative paths?

**Comment by dai-shi** [comment]
Hmm, adding extension broke the website.
@roguesherlock Is it possible to remove `.mdx` extension when deploying website?

**Comment by roguesherlock** [comment]
@dai-shi Currently the renderer we use to parse the markdown doesn't support it but It should be possible to add a plugin to do that for us.

**Comment by dai-shi** [comment]
> @dai-shi Currently the renderer we use to parse the markdown doesn't support it but It should be possible to add a plugin to do that for us.

Oh, that's unfortunate. I wonder if anyone would be interested in supporting it.
Otherwise, we would need to remove the extension for now...

**Comment by devagja** [comment]
I am going to change the routes as at the beginning.
I can research how to fix it but if anyone else can investigate, I'd appreciate it.

**Comment by dai-shi** [comment]
> I am going to change the routes as at the beginning.

If it's about a workaround, can it be relative paths without `.mdx`? It won't fix, but people can add the extension when 404.

**Comment by devagja** [comment]
> If it's about a workaround, can it be relative paths without `.mdx`? It won't fix, but people can add the extension when 404.

sorry for the lapse of time,
I just added the relative paths without the `.mdx`

**Review by dai-shi — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**